### ISSUE TITLE
Issue using tags

### ISSUE THREAD

**Issue by weyrougham10** [kahill2014/Hmong-Project#5]
Anybody understand how to use tags for photos. Tags should be single words. I used an input box for adding tags but will people be able to search by individual words from that data field? Sorry I don't really understand how to explain this. Just upload an image and fill all the fields in. Then check the sql and see how its put into the database. Basically will we be able to separate each individual words from tags and search using that.

**Comment by paulknisely** [comment]
Yes. If you input text and select "tags" from the drop down, it will search through that column of the database and return any result that is "LIKE" that input text. Of course, tags is not currently an option to search but that can be a quick add.

**Comment by paulknisely** [comment]
The only question I have is, how to separate each tag into a separate field within that tags input box.

**Comment by weyrougham10** [comment]
I know that there is a way to use tokenfields. When you hit space/enter it will create a tag.
Use this link to see what I mean. Problem is I would need to add another library to the pageheader but idk how to add the library via server so it works live.
http://sliptree.github.io/bootstrap-tokenfield/

**Comment by weyrougham10** [comment]
If it becomes an issue, you can ditch tags and I can create a input box for camp name. Be much easier. tags are kind of like "#" on twitter or instagram and i dont even know if thats appropriate for a site like this.
